docs/dev-manuals/architecture/requirements.md

```diff
@@ -173,7 +174,7 @@ This document is in the making since we are still in the requirements engineerin
 
 ##### Open Questions
 
-- What is klee frontend? The examiner mentioned it.
+- ~~What is klee frontend? The examiner mentioned it.~~
```

Comment:
Why keep the question striked-through? We could also remove it completely.

Reply:
Idk, I think it's generally best to not hard delete everything all the time, but I see your point. I will simply remove it.